bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/capability/Capability.java

```diff
@@ -113,6 +114,8 @@ protected void updateNAThing(NAThing newData) {
         }
         if (!newData.isReachable()) {
             statusReason = "@text/device-not-connected";
+        } else if (!newData.hasFreshData(DATA_AGE_LIMIT)) {
```

Comment:
This seems to override `isReachable()` after 30 minutes. I tested additional indoor module being offline a few days ago, and I think it was a little slower (not much) to go offline. Am I right that this module will now be marked offline before Netatmo services will actually consider it offline? If yes, can we add logic so that devices for which we can trust `isReachable()`, i.e. where we actually receive this information from the API, will not be overridden? Or alternatively set `DATA_AGE_LIMIT` higher. @lolodomo, you might also have some insights here.

Reply:
To perhaps clarify a bit, what I mean is that if for example weather station indoor/outdoor modules provide this information, we should use it and not be overridden by the hardcoded expiration limit. Perhaps it could be implemented by a `supportsReachable()` or something similar defaulting to `false`. Logic would then be something like (only for explanation purpose, not actual implementation):

```
if (newData.supportsReachable()) {
    if (!newData.isReachable()) {
        statusReason = "@text/device-not-connected";
    }
} else if (!newData.hasFreshData(DATA_AGE_LIMIT)) {
    statusReason = "@text/data-over-limit";
}
```

Reply:
@jlaur : the problem is that the "reachable" field seems to not work at least for the main weather station. During my tests last days, I see that after unplugging my weather station, the API still returns that the station was reachable even 1 hour after I unplugged it !
I have not checked the reachable field for additional modules.

Reply:
My additional indoor module became unreachable (thus **OFFLINE**) in less than one hour, so this seems to work. Most likely this behavior is similar to the outdoor module which you also unintentionally tested. :) So for these modules it seems we can use the API without any overridden logic.

Reply:
For the outdoor module, I will check tomorrow if "reachable" is set to false by the API.
In case the API works as expected for additional modules, we could decide to use `hasFreshData` only for the main weather station. I will also run a new test with the weather station and wait several hours to see if the API updates or not the "reachable" field.

Reply:
I can check the API response, but is must be included as otherwise it wouldn't have been marked as offline automatically before the data age limit was implemented. For the main weather station it makes some sort of sense that this information is not available, since this can be for various reasons (Wi-Fi, LAN, router, ISP). However, main station is directly responsible for its additional modules, so when RF communication is not working, main station is the correct device to detect this.

Reply:
It works correctly now - 17:18:55 batteries out:

```
2022-05-06 17:55:06.672 [INFO ] [ab.event.ThingStatusInfoChangedEvent] - Thing 'netatmo:indoor:home:indoor_small_bathroom:indoor_office' changed from ONLINE to OFFLINE: Device is not connected
2022-05-06 18:45:33.247 [INFO ] [ab.event.ThingStatusInfoChangedEvent] - Thing 'netatmo:indoor:home:indoor_small_bathroom:indoor_office' changed from OFFLINE: Device is not connected to ONLINE

```